src/System.Data.Common/tests/System/Data/Common/DbDataReaderMock.cs

```diff
@@ -90,12 +84,21 @@ public override long GetBytes(int ordinal, long dataOffset, byte[] buffer, int b
 
         public override char GetChar(int ordinal)
         {
-            throw new NotImplementedException();
+            return (char)GetValue(ordinal);
         }
 
         public override long GetChars(int ordinal, long dataOffset, char[] buffer, int bufferOffset, int length)
         {
-            throw new NotImplementedException();
+            object value = GetValue(ordinal);
```

Comment:
This seems to be going beyond what a mock object should be implementing, and I'm not sure it has much value, i.e. just for checking that the string overloads do indeed delegate to the int overloads. But it's no big deal either.

Reply:
@roji, I didn't quite understand what you meant here. Do you mean testing this change is not needed?

Reply:
In general I do think the test value here is a bit limited - but let's leave the code as it is.

Reply:
As above.